boat-scaffold/src/main/java/com/backbase/oss/codegen/angular/BoatAngularGenerator.java

```diff
@@ -222,14 +222,20 @@ private void addDependencies(SemVer angularVersion) {
             supportingFiles.add(new SupportingFile("ng-package.mustache", getIndexDirectory(), "ng-package.json"));
             supportingFiles.add(new SupportingFile("tsconfig.mustache", getIndexDirectory(), "tsconfig.json"));
             additionalProperties.put("zonejsVersion", "0.10.2");
```

Comment:
@santam85 can we just use 1 version of rxjs which is in this case 0.11.7 ?

Reply:
I guess you mean zone.js? let's use the default version that gets generated with `ng new` for each Angular major, just to be sure...